pkg/dhcpcd/Dockerfile

```diff
@@ -16,4 +16,4 @@ WORKDIR /
 COPY --from=mirror /out/ /
 COPY /dhcpcd.conf /usr/ /
 CMD ["/sbin/dhcpcd", "--nobackground", "-f", "/dhcpcd.conf"]
-LABEL org.mobyproject.config='{"binds": ["/var:/var", "/tmp/etc:/etc"], "capabilities": ["CAP_NET_ADMIN", "CAP_NET_BIND_SERVICE", "CAP_NET_RAW"]}'
+LABEL org.mobyproject.config='{"binds": ["/run/resolvconf:/etc"], "capabilities": ["CAP_NET_ADMIN", "CAP_NET_BIND_SERVICE", "CAP_NET_RAW"]}'
```

Comment:
I think we are talking at cross purposes, all I was suggesting was that the `pkg/dhcpcd/Dockerfile` have in it:
```
RUN ln -nfs /run/resolvconf/resolve.conf /etc/resolv.conf
LABEL org.mobyproject.config='{"binds": ["/run/resolvconf:/run/resolvconf"], ...
```
Then the dhcpc client writes through it's local `/etc/resolv.conf` into the host/shared `/run/resolvconf/resolve.conf` which the host then picks up via it's own symlink. Other "normal" packages just bind `/etc/resolv.conf:/etc/resolv.conf` as normaly and rely on the symlink deref behaviour of `runc` to get the right thing.